packages/components/src/tools-panel/test/index.tsx

```diff
@@ -21,49 +22,49 @@ const defaultProps = {
 };
 
 // Default props for an enabled control to be rendered within panel.
+let controlValue: boolean | undefined = true;
 const controlProps = {
-	attributes: { value: true },
 	hasValue: jest.fn().mockImplementation( () => {
-		return !! controlProps.attributes.value;
```

Comment:
This self-referencing was quite awkward. Thanks for fixing those!

Reply:
Yeah — I appreciate that it's self contained, but it's a bit akward and prone to breakage (and in fact TS doesn't like it)

Reply:
I'm in general not a fan of this "shared state" patten across tests, but we have more pressing priorities at the moment

Reply:
Yeah, not a huge deal and something we can easily address if needed.